package.json

```diff
@@ -56,9 +59,11 @@
     "webpack-stream": "^3.1.0"
   },
   "build": {
-    "linux": {
-      "synopsis": "Mattermost Desktop"
-    }
+  "app-bundle-id": "org.mattermost.desktop",
```

Comment:
.com please since desktop apps will eventually be hosted at releases.mattermost.com, sorry for late response for some reason I didn't see this mention